packages/sushi/src/config/squid.ts

```diff
@@ -0,0 +1,70 @@
+import { ChainId } from '../chain'
+
+export const SquidIntegratorId = 'sushiswap-sdk'
+
+export const SQUID_CHAIN_NAME = {
```

Comment:
is this coming from squid directly? no issues it's that

just found it weird that some start upper some lower